packages/firestore/package.json

```diff
@@ -7,14 +7,15 @@
   "description": "The Cloud Firestore component of the Firebase JS SDK.",
   "author": "Firebase <firebase-support@google.com> (https://firebase.google.com/)",
   "scripts": {
+    "api-report:lite": "(cd lite; api-extractor run --local --verbose)",
```

Comment:
Once this [PR](https://github.com/microsoft/rushstack/pull/1932) is merged, we can generate api reports for multiple entry points together, so you won't generate report for lite and main separately.
It seems it might take a long time to merge though, so I might publish a forked version to npm which we will use. For now, this is okay.